util/operations/utils.go

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
this file seems to be azure specific. i'd suggest renaming the package name accordingly.

Reply:
renamed to util/azure/utils.go